feat(outcomes): Add documentation for new information on outcome "Too Large". (#13012)
getsentry/sentry-docs: docs/product/stats/index.mdx

- **Project Missing**: The specified Sentry project is disabled or project information in the submitted request was inconclusive.
- **Sampling**: Tracing data that is discarded during sampling.
- **Security Report**: An invalid or unsupported security report was submitted. If this report originates from an up-to-date browser, please report a bug to Sentry. See [Security Policy Reporting](/security-legal-pii/security/security-policy-reporting/) for more information.

> - **Too Large**: The HTTP submission or its contained data exceeds size limits. The reason will be of the form "Too Large <reason>" where reason indicates which specific size limit was exceeded (for example, "Too Large Attachment"). See [this document](https://develop.sentry.dev/sdk/data-model/envelopes/#size-limits) for the precise limits.

Reviewer: ```suggestion
- **Too Large**: The HTTP submission or its contained data exceeds size limits. The reason will be given as "Too Large <reason>"; the 'reason' will tell you which specific size limit was exceeded (for example, "Too Large Attachment"). See [this document](https://develop.sentry.dev/sdk/data-model/envelopes/#size-limits) for the precise limits.
```

> - **Too Large**: The HTTP submission or its contained data exceeds size limits. The reason will be of the form "Too Large <reason>" where reason indicates which specific size limit was exceeded (for example, "Too Large Attachment"). See [this document](https://develop.sentry.dev/sdk/data-model/envelopes/#size-limits) for the precise limits.

Reviewer 2: ```suggestion
- **Too Large**: The HTTP submission or its contained data exceeds size limits. If available, what exactly exceeded might be provided as "Too Large <what>", where <what> specifies the specific size limit that was exceeded (e.g., "Too Large Attachment"). For the exact size limits, refer to [this document](https://develop.sentry.dev/sdk/data-model/envelopes/#size-limits)
```

Reviewer 2: we already call 'Too Large' a reason, so I would use the word "what" instead of "reason"again

Reviewer 2: or type instead!
